zeppelin-web/app/scripts/controllers/paragraph.js

```diff
@@ -912,11 +918,11 @@ angular.module('zeppelinWebApp')
 
       var chartEl = d3.select('#p'+$scope.paragraph.id+'_'+type+' svg')
           .attr('height', $scope.paragraph.config.graph.height)
-          .style('height', height + 'px')
```

Comment:
This part was actually from a new commit, and should not be removed, my guess is that your branch is not up to date with master.